src/components/ImageUploader.js

```diff
@@ -155,26 +186,41 @@ class ImageUploader extends React.Component {
     }
   };
 
-  handleImageUploaderChange = (element, e, newValue) => {
+  handleImageUploaderChange = (element, e, newValue, imageIndex) => {
     switch (element) {
       case constants.IMAGE_PATH_IDENTIFIER: {
-        let file = e.target.files[0];
-        let imageLabel = file.name.split('/').pop();
-        let imageName = imageLabel.split('.');
-        let extension = imageName.pop().toLowerCase();
-
-        if (constants.ACCEPTED_IMAGE_FORMATS.indexOf(extension) === -1) {
-          this.vars.snackbarWarning =
-            constants.PROPER_FORMAT_ACCEPTED_WARNING +
-            constants.ACCEPTED_IMAGE_FORMATS;
-          this.setState({ shouldDisplayWarningSnackBar: true });
-          break;
+        let files = e.target.files;
+        let imageLabels = '';
+        let imageNames = [];
+        let paths = [];
+        for (let i = 0; i < files.length; i++) {
+          let imageLabel = files[i].name.split('/').pop();
+          let imageName = imageLabel.split('.');
+
+          let extension = imageName.pop().toLowerCase();
+          imageLabels += imageLabel;
+          if (i < files.length - 1) {
+            imageLabels += ',';
+          }
+          paths.push(files[i].name);
+          imageNames.push(imageName[0]);
```

Comment:
You need to do a join here.

Reply:
I stored imageNames(not contain extension) as a imageName array contains all the information of image names. And I don't need the extension until I upload the images. So I  didn't do the join here.

Reply:
What if the image has more than one dots? 'a.b.jpg'? There are some cases of this that is the reason I had this join.